previewer/presentationvideo.js

```diff
@@ -91,11 +91,13 @@
                 slide_image = document.createElement("IMG");
                 slide_image.setAttribute("src", jsRoutes.api.Previews.download(elem[2]).url);
                 slide_image.setAttribute("title", "Click/tap on this slide to navigate to it in the video");
+                slide_image.setAttribute("alt", "Slide " + (index+1));
```

Comment:
Since we know the total count, can we make this:
```
slide_image.setAttribute("alt", "Slide " + (index+1) + " of " + extract_data[0][0]['content']['nrslides']);
```